### PR TITLE
578 cross sell links

### LINKED ISSUE #578: Cross-referenced life event links break after page reload
Cross-referenced life event links don't work properly after page reload. To replicate:

- From Death of a loved one life event, click on Retirement life event card at the bottom of the page
- On Retirement life event, reload the page and click on Death of a loved one life event card

See: https://www.screencast.com/t/tnYGvS8WDl0r

### PR DESCRIPTION
**Pull request by fpigeonjr**
closes #578

### CODE REVIEW

**Comment by fpigeonjr** [comment]
😄 cross-sell links work in english and in español

**Comment by fpigeonjr** [comment]
🛑  will need to revert back to `single-life-event-mode` once it is reviewed

File: content/landing-page.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: landing-page.title
 summary: landing-page.summary
-lifeEvent: death-of-a-loved-one
```
Comment (fpigeonjr) [inline content/landing-page.md:4]: 🛑 will need to revert once this is reviewed.

**Review by FatmaBakir — APPROVED** [review]

File: content/landing-page.md (hunk 1 above)
Comment (fpigeonjr) [reply content/landing-page.md]: its back

**Pull request merged**